language/tools/move-package/src/source_package/parsed_manifest.rs

```diff
@@ -55,15 +56,31 @@ pub struct InternalDependency {
     pub subst: Option<Substitution>,
     pub version: Option<Version>,
     pub digest: Option<PackageDigest>,
+    pub dep_override: Option<DepOverride>,
```

Comment:
nit: This doesn't need to be an `Option<DepOverride>` we can just have it be a `bool` and parse `None` as `false` (and display `false` as no field, and `true` as `override = true`).

Also, I don't feel strongly about this, but I found the alias of `DepOverride = bool` a little confusing while reading -- I was expecting a bool but found what I thought was a struct, and was left wondering what other information it needed to carry.

Reply:
Makes sense!